soap/client.go

```diff
@@ -224,7 +224,7 @@ type Envelope struct {
 	XMLName      xml.Name `xml:"SOAP-ENV:Envelope"`
 	EnvelopeAttr string   `xml:"xmlns:SOAP-ENV,attr"`
 	NSAttr       string   `xml:"xmlns:ns,attr"`
-	TNSAttr      string   `xml:"xmlns:tns,attr"`
+	TNSAttr      string   `xml:"xmlns:tns,attr,omitempty"`
```

Comment:
It will be rewritten on https://github.com/fiorix/wsdl2go/pull/97/files#diff-3bb06cfa05708bb86d2eec5ee5ce82b0R110

Reply:
How that would be rewritten, sorry? You are adding `URNAttr`, but not removing `TNSAttr` or modifying its attributes

Reply:
In the same file. On line 110 you have check
```
if req.TNSAttr == "" {
	req.TNSAttr = req.NSAttr
}
```